packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -95,6 +95,20 @@ export const ItemBadgeUI = styled.span`
 	display: inline-flex;
 	padding: 4px 12px;
 	border-radius: 2px;
+	animation: fade-in 250ms ease-out;
+
+	.components-spinner {
+		margin: 0;
+	}
+
+	@keyframes fade-in {
+		from {
+			opacity: 0;
+		}
+		to {
+			opacity: 1;
+		}
+	}
```

Comment:
We have a mixin for fade-ins: [`edit-post__fade-in-animation`](https://github.com/WordPress/gutenberg/blob/50955c1f8ee632a34a8faea2b22bf62fbaefae5a/packages/base-styles/_animations.scss).
The name is counter-intuitive, but I've see it used for components as well.

I'm not familiar enough with styled components to know if it's possible to import a SASS mixin.
If we can't, I think we should at least use the allow folks to [opt out of animations](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) with [`reduce-motion`](https://github.com/WordPress/gutenberg/blob/50955c1f8ee632a34a8faea2b22bf62fbaefae5a/packages/base-styles/_mixins.scss#L214-L239).
We even have a handy [JS utility](https://github.com/WordPress/gutenberg/blob/50955c1f8ee632a34a8faea2b22bf62fbaefae5a/packages/components/src/utils/reduce-motion.js) for that already! ✨ 

I think it should be used like this:
```js
const ItemBadgeUI = styled.span`
  animation: fade-in 250ms ease-out;
  @keyframes fade-in { /* ... */ }

  ${ reduceMotion( 'animation' ) };
`;
```

Reply:
We can't use SASS in Styled :( How about creating a styles file? We already have a class for the badge: 
```scss
.components-navigation__item-badge {
	@include edit-post__fade-in-animation;
}
```

Later on we should recreate this mixin in JS though so we can use it with Styled.

Reply:
I think we should follow Gutenberg's own direction, which unfortunately I don't know. 😅 

If we're staying with SASS, then sure let's add a `style.scss` file!
If we're moving to styled components, then let's go with a fade-in JS mixin!

Either way, I think in this specific case it's fine to keep what you did, and just add the `${ reduceMotion( 'animation' ) }`.